### DISCUSSION TITLE
Excel raw compare

### DISCUSSION THREAD

**Discussion by IgorST91** [community/community#21508] [Other Feature Feedback, Questions, & Ideas]
<p>Hello. There is excel file with 5 columns and 1000 raws. Looking for a code that will compare 2 columns. It is columns with database logins and LDAP logins. If value of column 1 differ of column 2 - paint red colour this 2 column. will be nice to find this code on github.</p>

**Comment by logankilpatrick** [comment, marked as answer]
<p>Hey <a class="mention" href="/igorst91">@igorst91</a>, this post describes a pretty good solution: <a href="https://stackoverflow.com/questions/43970265/excel-columns-comparison-using-python-code" rel="nofollow noopener">https://stackoverflow.com/questions/43970265/excel-columns-comparison-using-python-code </a></p>
<p>I am not sure if Panda’s has the ability to write colored columns but that would be trivial to do manually. I hope this helps! </p>
